mdoc/Resources/monodoc-ecma.xsd

```diff
@@ -481,11 +481,23 @@ add masterdoc support?
     </xs:complexType>
   </xs:element>
 
+  <xs:element name="MemberGroup">
+    <xs:complexType>
+      <xs:choice minOccurs="0" maxOccurs="unbounded">
+        <xs:element ref="AssemblyInfo" minOccurs="0" maxOccurs="unbounded" />
+        <xs:element ref="summary" minOccurs="0" maxOccurs="1" />
```

Comment:
Should `MemberGroup` instead include `Docs`? This would make `MemberGroup` more similar to `Member`, and I *suspct* would simplify the XSLT for HTML rendering -- which I notice that this PR is lacking.